src/main/java/de/adesso/service/JekyllService.java

```diff
@@ -20,45 +19,44 @@
     private final ConfigService configService;
 
     @Autowired
-	public JekyllService(ConfigService configService) {
-		this.configService = configService;
-	}
+    public JekyllService(ConfigService configService) {
+        this.configService = configService;
+    }
 
-	/**
+    /**
      * Starts the jekyll build process (jekyll build --incremental)
      */
     public void startJekyllBuildProcess() {
-		LOGGER.info("Starting jekyll build");
-		// create command builder
-		ProcessBuilder builder = new ProcessBuilder();
-		// set command:
-		//   execute in shell
-		//   allow file access of repo dir for user jekyll
-		//   call jekyll build
-		builder.command("sh", "-c", "chown -R jekyll /srv/jekyll/repo && jekyll build --incremental");
-		LOGGER.info("Builder working dir: " + configService.getLOCAL_REPO_PATH());
-		// set dir where to execute command
-		builder.directory(new File(configService.getLOCAL_REPO_PATH()));
-
-		Process process = null;
-		try {
-			LOGGER.info("execute command: sh -c chown -R jekyll /srv/jekyll/repo && jekyll build --incremental");
-			process = builder.start();
-		} catch (IOException e) {
-			e.printStackTrace();
-		}
-		// pass every out line from process to sysout
-		StreamGobbler streamGobbler = new StreamGobbler(process.getInputStream(), System.out::println);
-		Executors.newSingleThreadExecutor().submit(streamGobbler);
-		int exitCode = 0;
-		try {
-			exitCode = process.waitFor();
-			LOGGER.info("jekyll build finished with exit code: " + exitCode);
-		} catch (InterruptedException e) {
-			e.printStackTrace();
-		}
-		if (exitCode != 0) {
-			System.exit(20);
-		}
+        LOGGER.info("Starting jekyll build...");
+        // create command builder
+        ProcessBuilder builder = new ProcessBuilder();
+        // set command:
+        //   execute in shell
+        //   allow file access of repo dir for user jekyll
+        //   call jekyll build

```

Comment:
Kommenare sollten nur das warum erklären oder fachliche Zusammenhänge erklären. In diesem Fall: Sehr guter Einsatz eines Kommentars! 👍  :)